uaclient/api/tests/test_u_pro_attach_auto_full_auto_attach_v1.py

```diff
@@ -0,0 +1,58 @@
+import mock
+import pytest
+
+from uaclient.api import exceptions
+from uaclient.api.u.pro.attach.auto.full_auto_attach.v1 import (
+    FullAutoAttachOptions,
+    full_auto_attach,
+)
+
+M_API = "uaclient.api.u.pro."
+
+
+class TestFullAutoAttachV1:
```

Comment:
Let's add tests for all the exceptions that we can raise in the API, just to be sure everything is working as expected